src/modules/auth/clients/ClientDetailContainer.js

```diff
@@ -164,15 +164,38 @@ const ClientDetailContainer = (props) =>  {
 				<CardBody>
 					<Row>
 						<Col md={5} title="login_uri">{t("ClientDetailContainer|Login URI")}</Col>
-						<Col>{client?.login_uri ? client.login_uri : "N/A"}</Col>
+						<Col>{client?.login_uri ? client.login_uri : "-"}</Col>
 					</Row>
 					<Row>
-						<Col md={5} title="cookie_domain">{t("ClientDetailContainer|Cookie domain")}</Col>
-						<Col title="cookie_domain">{client?.cookie_domain ? client.cookie_domain : "N/A"}</Col>
+						<Col md={5} title="authorize_uri">{t("ClientDetailContainer|Authorize URI")}</Col>
+						<Col>{client?.authorize_uri ? client.authorize_uri : "-"}</Col>
+					</Row>
+				</CardBody>
+			</Card>
+
+			<Card className="client-cookie-card">
+				<CardHeader className="border-bottom">
+					<div className="card-header-title">
+						<i className="cil-paperclip pr-2"></i>
```

Comment:
@byewokko Just a curious question - shouldnt be this icon something like `cookie`? https://coreui.io/icons/linear/blur-circular/

When there will be a transition to atlas icons done ,I propose `at-cookie` @petrKavulok

Reply:
that might be a better idea. i'll change it.